### ISSUE TITLE
how it works

### ISSUE THREAD

**Issue by saman-aliveisi** [miguelmazetto/PocketPC#2]
hi friend , Please explain how it works (take video demo)
thanks

**Comment by miguelmazetto** [comment]
Im working on the backend, it's still on WIP. But thanks for the interest, in the future it should be one click and you have access to steam proton or something like that

**Comment by saman-aliveisi** [comment]
thanks bro.
i want make application for android like exageaer that run wine on android.
How to add wine in this project?

**Comment by miguelmazetto** [comment]
Yes It Will have It, in fact, i have already confirmed the possibility of everything here.
Here`s a tip, Box86Launcher already implemented box86 -> wine x86, It Works but has a poor x server and no gpu acceleration.
This project will do:
rwroot (my alternative to proot) -> wine arm64 -> WOW64 -> box64 -> wine x86_64
With this it should have the best performance possible without root.

**Comment by saman-aliveisi** [comment]
this is awsome.
im running repo and have questions.
how i add wine to repo ? Is it possible to add wine obb?
I would be grateful if you could provide a demo that load wine .

![image](https://user-images.githubusercontent.com/9861867/194000228-441056e8-9f52-4dc6-8fc4-3074d483c59d.png)

**Comment by miguelmazetto** [comment]
As I said, WIP (work in progress), still not working
Go look for Box86Launcher
